tests/SprykerSdkTest/Spryk/Integration/Glue/Dependency/Client/AddGlueDependencyClientBridgeMethodTest.php

```diff
@@ -136,8 +123,6 @@ public function testAddsGlueDependencyClientBridgeMethodOnlyOnce(): void
      */
     public function testAddsGlueDependencyClientBridgeMethodOnlyOnceOnProjectLayer(): void
     {
-        $this->expectException(SprykWrongDevelopmentLayerException::class);
-
         $this->tester->run($this, [
```

Comment:
This is the now failing line, being cloaked by the wrong position of expectException() which should only happen to 2nd run() call.

Reply:
@AsonUnique Can you please elaborate?

Reply:
It seems that you are right, the `expectException()` call is wrongly positioned.

Reply:
Now it is not wrongly positioned anymore, as mentioned above.
It now shows an issue in code, though, that needs fixing.
Do you know why it fails (wrongly)?

Reply:
I'm afraid it was too long ago and I hardly remember what is going on here. 
Maybe @limeeugenia could help you here?

Reply:
For me it looks like test name was wrong. IMO it tests that we can't add core related code on project level

Reply:
Check the other 3-5 places where the same code is executed and the tests seem to pass
I do agree that the message here seems confusing.

Reply:
Yep. But there is a `testAddsGlueDependencyClientBridgeMethodsOnProjectLayer` test that tests actually this case.
So IMO we can remove this failing test at all.

Reply:
Fair enough.